schema/rss-service-plan-v3.json

```diff
@@ -18,6 +18,99 @@
     }
   },
   "definitions": {
+    "DatasetConfig": {
```

Comment:
Great catch.

Although I believe this change is additive, I'm going to be conservative, and just update this to `rss-service-plan-v5.json` -- this preserves the past formats, and diffing `v4` and `v5` shows that this change is purely an addition.

I've also filed https://github.com/oxidecomputer/omicron/issues/6522 for later, because I *think* we don't need to read this file after initial bootstrapping, and perhaps that should be encoded more explicitly.